generateShopItems.js

```diff
@@ -0,0 +1,36 @@
+import { fetchData } from './getData.js'
+
+function generateShopItem(item) {
+	const shopList = document.getElementById('shopList')
+
+	const shopItem = `
+  <li class="items-container__item">
+  <div class="photo">
+    <img
+      class="item"
+      src="${item.url}"
+      alt="Shop item"
+    />
+  </div>
+  <div class="information">
+    <p class="description">${item.title}</p>
+    <p class="price">${item.price}</p>
+  </div>
+</li>
+  `
+
+	shopList.insertAdjacentHTML('beforeend', shopItem)
+}
+
+export async function createShopItemList() {
```

Comment:
The function name implies that this function creates items, but rather it also fetches the needed data. Consider either renaming the function or splitting the logic. 😉